scripts/update_meilisearch_version.sh

```diff
@@ -212,63 +211,57 @@ cp meilisearch /usr/bin/meilisearch
 systemctl restart meilisearch
 echo "${INFO_LABEL}MeiliSearch $meilisearch_version is starting."
 
-if systemctl status meilisearch | grep -E 'Active: active \(running\)' -q; then
-    echo "${INFO_LABEL}Both MeiliSearch versions are compatible. No need to dump."
-else
-    echo "${INFO_LABEL}MeiliSearch versions are not compatible with each other."
-
-    # Stopping MeiliSearch Service
-    systemctl stop meilisearch
-    echo "${INFO_LABEL}Stop MeiliSearch $meilisearch_version service."
+# Stopping MeiliSearch Service
+systemctl stop meilisearch
+echo "${INFO_LABEL}Stop MeiliSearch $meilisearch_version service."
 
-    # Keep cache of previous data.ms in case of failure
-    cp -r /var/lib/meilisearch/data.ms /tmp/
-    echo "${INFO_LABEL}Copy data.ms to be able to recover in case of failure."
+# Keep cache of previous data.ms in case of failure
+cp -r /var/lib/meilisearch/data.ms /tmp/
+echo "${INFO_LABEL}Copy data.ms to be able to recover in case of failure."
 
-    # Remove data.ms
-    rm -rf /var/lib/meilisearch/data.ms
-    echo "${INFO_LABEL}Delete current MeiliSearch's data.m."
+# Remove data.ms
+rm -rf /var/lib/meilisearch/data.ms
+echo "${INFO_LABEL}Delete current MeiliSearch's data.ms"
```

Comment:
Why keep these steps if Meilisearch will be restarted with a dump? Included from the restart line 211.
If I restart Meiliserach with the `data.ms` and the dump it will not re-index all my data?

Reply:
It will! The purpose of this PR si to not use the old `data.ms` anymore unless we rollback to the previous version. 
Removing the `data.ms` is usefull as it may cause the new binary to throw on launch. 

We also need to stop the service to replace the binaries